modules/sbo-setting-annotations-mapping-optional.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: PROCEDURE
+[id="sbo-setting-annotations-mapping-optional_{context}"]
+= Setting annotations mapping to be optional
+
+You can have optional fields in the annotations. For example, a link to the credentials might not be there if the service endpoint does not require authentication. In such cases, a field might not exist in the target path of the annotations. As a result, {servicebinding-title} generates an error, by default. 
+
+As a service provider, to indicate whether you require annotations mapping, you can set a value to the `optional` flag in your annotations when enabling services. {servicebinding-title} provides annotations mapping only if the target path is available. When the target path is not available, the {servicebinding-title} skips providing the annotations mapping. 
```

Comment:
I would say "skips the optional mapping and continues with the projection of the existing mappings without throwing any errors" something like that. The wording "skips providing the annotations mapping" is a bit ambiguous in my opinion.

Reply:
Thanks, @dperaza4dustbit! This looks much better :)

Reply:
Done.